lib/utils/createModifier.js

```diff
@@ -1,12 +1,5 @@
 const { asArray, isString, isFunction, isPlainObject } = require('./objectUtils');
 
-class ModifierNotFoundError extends Error {
-  constructor(mofifierName) {
-    super(`Unable to determine modify function from provided value: "${mofifierName}".`);
-    this.mofifierName = mofifierName;
```

Comment:
@koskimas I've spotted a spelling mistake here: `mofifierName`. I've corrected this in the new location of the class.